internal/cli/internal/command/volumes/snapshots/list.go

```diff
@@ -0,0 +1,75 @@
+package snapshots
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	"github.com/dustin/go-humanize"
+	"github.com/spf13/cobra"
+
+	"github.com/superfly/flyctl/pkg/iostreams"
+
+	"github.com/superfly/flyctl/internal/cli/internal/command"
+	"github.com/superfly/flyctl/internal/cli/internal/config"
+	"github.com/superfly/flyctl/internal/cli/internal/flag"
+	"github.com/superfly/flyctl/internal/cli/internal/render"
+	"github.com/superfly/flyctl/internal/client"
+)
+
+func newList() *cobra.Command {
+	const (
+		long  = "List snapshots associated with the specified volume"
+		short = "List snapshots"
+
+		usage = "list <volume-id>"
+	)
+
+	cmd := command.New(usage, short, long, runList,
+		command.RequireSession,
+	)
+
+	cmd.Args = cobra.ExactArgs(1)
+
+	return cmd
+}
+
+func runList(ctx context.Context) error {
+	var (
+		io     = iostreams.FromContext(ctx)
+		cfg    = config.FromContext(ctx)
+		client = client.FromContext(ctx).API()
+	)
+
+	volID := flag.FirstArg(ctx)
+
+	snapshots, err := client.GetVolumeSnapshots(ctx, volID)
+	if err != nil {
+		return err
```

Comment:
this error should be decorated as: `fmt.Errorf("failed retrieving snapshots: %w", err)`

Reply:
Should I decorate all `runSomeCmd` errors this way?